app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -46,6 +46,9 @@ XrResult OpenXRInputSource::Initialize()
     RETURN_IF_XR_FAILED(mActionSet.GetOrCreateAction(XR_ACTION_TYPE_POSE_INPUT, "grip", OpenXRHandFlags::Both, mGripAction));
     RETURN_IF_XR_FAILED(mActionSet.GetOrCreateAction(XR_ACTION_TYPE_POSE_INPUT, "pointer", OpenXRHandFlags::Both, mPointerAction));
 
+    // Initialize haptic action.
```

Comment:
Remove this comment too. Also put the line bellow right next to the other `GetOrCreateAction`.